esmvaltool/cmorizers/data/cmor_config/ESACCI-WATERVAPOUR.yml

```diff
@@ -1,34 +1,46 @@
 ---
 # Common global attributes for Cmorizer output
 
-
-# Input
-# CDR-2
-filename: 'ESACCI-WATERVAPOUR-L3S-TCWV-*-05deg-{year}{month}-fv3.1.nc'
-# CDR-1
-# filename: 'dataset3_1/CDR-1/monthlies/ESACCI-WATERVAPOUR-L3?-TCWV-*-05deg-{year}{month}-fv3.1.nc'
 attributes:
   dataset_id: ESACCI-WATERVAPOUR
   # CDR-2
-  version: 'CDR2-L3S-05deg_fv3.1'
+  version: CDR2_L3C-05deg_fv3.1
   # CDR-1
-  # version: 'CDR1-L3-05deg_fv3.1'
+  # version: CDR1-L3C-05deg_fv3.1
   tier: 3
   modeling_realm: sat
-  project_id: OBS
-  source: "ftp.brockmann-consult.de, access currently restricted"
+  project_id: OBS6
+  # CDR2
+  source: https://wui.cmsaf.eu/safira/action/viewDoiDetails?acronym=COMBI_V001
+  # CDR1
+  # source: https://data.ceda.ac.uk/neodc/esacci/water_vapour/data/TCWV-land/L3/v3.1/
   reference: ["esacci-watervapour"]
-  comment: "Preliminary data."
+  comment: "Data from different phases is distributed from different sources."
```

Comment:
Maybe something like: "Downloading is possible after a registration and ordering of the data."

Reply:
Corrected.